com.unity.render-pipelines.universal/Runtime/2D/Shadows/ShadowRendering.cs

```diff
@@ -32,20 +44,80 @@ public static void InitializeBudget(uint maxTextureCount)
             }
         }
 
+        private static Material GetProjectedShadowMaterial(this Renderer2DData rendererData, int colorMask)
+        {
+            //if (rendererData.projectedShadowMaterial == null || rendererData.projectedShadowShader != rendererData.projectedShadowMaterial.shader)
```

Comment:
Just double checking, do we want to create a new one every time the method is called? If so, maybe renaming to `CreateProjectedShadowMaterial` could make sense?

Reply:
Fixed